README.md

```diff
@@ -10,7 +10,7 @@ Chef Development Kit (ChefDK) brings Chef and the development tools developed by
 
 This repository contains the code for the `chef` command. The full
```

Comment:
since you're here, you can just delete this. omnibus-chef is now part of this repo